telegram/_message.py

```diff
@@ -3522,6 +3526,19 @@ def _parse_markdown(
                             "Spoiler entities are not supported for Markdown version 1"
                         )
                     insert = f"||{escaped_text}||"
+                elif entity.type == MessageEntity.CUSTOM_EMOJI:
+                    if version == 1:
+                        # this ensures compatibility to previous PTB versions
+                        insert = escaped_text
+                    else:
+                        # This should never be needed because ids are numeric but the documentation
+                        # specifically mentions it so here we are
```

Comment:
Which documentation exactly are we talking about? 🤔

Reply:
Telegrams ofc. About replacing chars in the () part of inline links and now custom emojis

Reply:
Would you mind adding a link to https://core.telegram.org/bots/api#formatting-options?

Reply:
I think it is clear where we get what to escape from, no?

Reply:
At least I got confused by the "are numeric" part, I expected a documentation about that … but nevermind